dev-manifests/service-account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
also sounds suitable to reside on ``tests/integration/manifests/`` dir

Reply:
I've moved them there, but to be fair I would expect to find only code there...
Maybe in the future we can better structure multiple manifests need.

Reply:
Well, kubernetes yamls are infrastructure-as-code in my opinion 😃

Reply:
I totally agree, but I meant code for the deployable `events_scrape` :sweat_smile: